packages/preview/clean-hda/0.1.0/README.md

```diff
@@ -0,0 +1,293 @@
+# Fork For Hochschule Darmstadt - University of Applied Sciences
+
+I recreated
+[mbredel/thesis-template](https://github.com/mbredel/thesis-template) in by
+forking the existing config of DHBW, which looked the most similar to the
+original latex implementation. 
+
+> [!NOTE]
+> This is an **unofficial** template for [Hochschule Darmstadt - University of Applied Sciences](www.h-da.de) for the department of Computer Sciences.
+> Contributions and takeover by h_da affiliated are welcome. 
+ 
+# How to include
+
+1. Add this as a git submodule
+```console
+git submodule add https://github.com/stefan-ctrl/clean-hda-typst-template hda_template
+```
+
+2. Include as the following in your `main.typst`:
+
+```typst
+#import "./hda_template/template/main.typ": *
+```
+
+Contributions are welcome.
+
+# Clean HDA
```

Comment:
You seem to be including multiple top-level headings in this readme. This may be displayed incorrectly on typst universe: https://github.com/typst/packages/blob/main/docs/documentation.md#differences-from-standard-markdown